templates/home.html

```diff
@@ -21,68 +21,75 @@ <h1 class="home-title">PERDIEM</h1>
         <h2 class="home-description">THE NEXT GENERATION OF MUSIC</h2>
         {% if user.is_authenticated %}
             <div class="home-boxes">
```

Comment:
Since both authenticated and unauthenticated users see the same thing here, there's no need for the `if` statement. We can just replace this with:
```
<div class="home-boxes">
    <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}">START INVESTING</a>
</div>
```

Also watch out, it looks like the quotes around the URL in the `a` tag are missing in your change.